chapters/101/index.md

```diff
@@ -1,167 +1,113 @@
 
-![Jina AI Executor, Copyright by Jina AI Limited](./img/ILLUS3.png)
 
-</span>
+### Crafter
 
+![Crafter Cartoon](images/crafter.svg)
 
-**Executor represents an algorithmic unit in Jina.** Algorithms such as encoding images into vectors, storing vectors on the disk, ranking results, can all be formulated as Executors. Executor provides useful interfaces, allowing AI developers and engineers to really focus on the algorithm. Features such as persistency, scheduling, chaining, grouping and parallelization come out of the box.
+Crafters pre-process input Documents, for example, resizing images or converting text to lower case. A Crafter often comes before the [Encoder](images/#encoder) but it’s not always required.
 
-The properties of an Executor are stored in a YAML config, they always go hands-in-hands.
 
+### Segmenter
 
-### Family of Executors
+![Segmenter Cartoon](images/segmenter.svg)
 
+Like a [Crafter](images/#crafter), a Segmenter also pre-processes Documents. A Segmenter breaks Documents into **multiple** chunks. For example, breaking a paragraph into sentences.
 
-<span style="display: inline-block; width: 60%;">
 
-![Jina 101 Family of Executor, Copyright by Jina AI Limited](./img/ILLUS4.png)
+### Encoder 
 
-</span>
+![Encoder Cartoon](images/encoder.svg)
 
+Encoders create vector embeddings of Documents, letting users perform semantic search. 
 
-**Executor is a big family.** Each family member focuses on one important aspect of the search system. Let’s meet:
-- **Crafter**: for crafting/segmenting/transforming the documents and chunks
-- **Encoder**: for representing the chunks as vectors
-- **Indexer**: for saving and retrieving vectors and key-value information from storage
-- **Ranker**: for sorting the results
 
-Having a new algorithm in mind? No problem, this family always welcome new members!
+### Indexer
 
+![Indexer Cartoon](images/indexer.svg)
 
-## Driver
-
-<span style="display: inline-block; width: 30%;">
+After Documents are [encoded](images/#encoder), an Indexer:
 
-![Jina 101 Driver, Copyright by Jina AI Limited](./img/ILLUS5.png)
+*   **Saves** Documents’ vector embeddings and metadata key-values pairs to storage (during indexing).
+*   **Retrieves** the vector embeddings and key-value pairs from storage (during querying).
 
-</span>
 
+### Ranker
 
-**Driver defines how an Executor behaves on network requests.** Driver helps the Executor to handle the network traffic by interpreting the traffic data (e.g. Protobuf) into the format that the Executor can understand and handle (e.g. Numpy array).
+![Ranker Cartoon](images/ranker.svg)
 
+Rankers sort and rank search results. They are frequently used in a querying Flow. 
 
 
+### Classifier
 
-## Pea
+![Classifier Cartoon](images/classifier.svg)
 
-<span style="display: inline-block; width: 30%;">
+Classifiers sort input Documents into categories and output the predicted hard/soft labels. Classifiers are optional, but may be useful depending on the use case.
```

Comment:
I think the verb sort here can be missleading, as it may confuse with Rankers

Reply:
@JoanFM Good point. I'm gonna `classify` for now since it's more straightforward.